sdk/storage/azure-storage-blobs/test/ut/append_blob_client_test.cpp

```diff
@@ -349,4 +349,30 @@ namespace Azure { namespace Storage { namespace Test {
     EXPECT_EQ(ReadBodyStream(appendBlobClient.Download().Value.BodyStream), blockContent);
   }
 
+  TEST_F(AppendBlobClientTest, OAuthAppendBlockFromUri_LIVEONLY_)
```

Comment:
I think we should update the sanitizer and make these test cases work in record-playback mode before merging this PR